### PR TITLE
Enhanced README: Contribution Guidelines, TOC, and Overview

### PR DESCRIPTION
**Pull request by batramahek**
Descriptive Overview: Added a detailed intro about Matrix.org, emphasizing its purpose.
Build Instructions: Provided detailed step-by-step instructions for building the website
Table of Contents: Added a table of contents to make navigation through the various sections easier
Contribution Guidelines: Created a clear section for contribution and added steps for how to start contributing to this open-source project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,51 @@
...
-## Building the website
 
-Install the latest version of [Zola](https://www.getzola.org), clone the repo, and from the root of the
-repo type
 
+## Building the website
+To contribute or run the website locally, follow the instructions below:
+1. '''
+git clone https://github.com/matrix-org/matrix.org.git
+cd matrix.org
```
Comment (MTRNord) [inline README.md:19]: This seems to not be working with Github Markdown. Could you fix that please? Have a look at the rich diff button on the top right of this file to see how it is being rendered

File: README.md [hunk 2]
```diff
@@ -1,21 +1,51 @@
...
 zola serve
 ```
 
 Zola will build the website and start a web server, usually at
 http://127.0.0.1:1111
 
+3. To build the website for production, run the command:
+'''
+zola build
+'''
```
Comment (MTRNord) [inline README.md:33]: Same as the other comment

File: README.md [hunk 3]
```diff
@@ -1,21 +1,51 @@
...
+
+## Contribution Guidelines
+1. Fork the reposity (https://github.com/matrix-org/matrix.org.git)
+2. Clone your fork to your local computer 
+3. Create a new branch and implement changes using
+'''
+git checkout -b <yourBranchName>
```
Comment (MTRNord) [inline README.md:40]: Same as the other comment

File: README.md [hunk 4]
```diff
@@ -1,21 +1,51 @@
...
+4. Submit a pull request (PR)
```
Comment (MTRNord) [inline README.md:41]:
```suggestion
4. Submit a pull request
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,51 @@
...
+
+
+# Contact Us
+
+## Community
+If you have any questions about the project or your contribution, you may join the various community chats on https://matrix.to/#/#matrix.org-website:matrix.org
```
Comment (MTRNord) [inline README.md:47]:
```suggestion
If you have any questions about the project or your contribution, you may join our room https://matrix.to/#/#matrix.org-website:matrix.org
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,51 @@
...
+- [Overview](#overview)
+- [Building the Website](#building-the-website)
+- [Contribution Guidelines](#contribution-guidelines)
+- [Contact Us](#contact-us)
 
-To discuss maintenance of this site, please come talk to the team in
-[#matrix.org-website:matrix.org](https://matrix.to/#/#matrix.org-website:matrix.org).
+# Overview
+
+Matrix.org is a decentralized, open standard for real-time communication which allows users to collaborate in public and private rooms over IP. Matrix's purpose is to allow users to text and call without thinking about what app or phone the other user has.
```
Comment (MTRNord) [inline README.md:11]:
```suggestion
Matrix.org is an open standard for decentralised, secure communications while also allowing you to bridge to other networks easily.

This Repository covers our main website.

For spec please take a look at https://github.com/matrix-org/matrix-spec
For the conference website take a look at https://github.com/matrix-org/matrix-conf-website
```

**Review by MTRNord — CHANGES_REQUESTED** [review]

**Comment by thibaultamartin** [comment]
OP didn't reply in 2 months on easy to address comments, and we highly suspect this was generated by a LLM.

**Pull request closed without merging**